docs/release_process.md

```diff
@@ -0,0 +1,269 @@
+are merged back to `main`:
+
+1. Create a branch from `main`
+2. Make changes
+3. Create a pull request to `main`
+4. Address review comments
+5. Merge pull request
+
+```mermaid
+%%{init: { 'theme': 'base', 'gitGraph': {'rotateCommitLabel': true} } }%%
+    gitGraph
+       commit
+       branch change-1
+       commit
+       commit
+       checkout main
+       merge change-1
+```
+
+This means many features could be in development at the same time, and all can
+merge back to `main` when they are ready.
+
+```mermaid
+%%{init: { 'theme': 'base', 'gitGraph': {'rotateCommitLabel': true} } }%%
+    gitGraph
+       commit
+       branch change-1
+       commit
+       commit
+       checkout main
+       branch change-2
+       commit
+       checkout main
+       merge change-1
+       branch change-3
+       commit
+       commit
+       checkout main
+       merge change-2
+       checkout change-3
+       commit
+       commit
+       checkout main
+```
+
+## Release Timeline
+The standard release interval for Relay is 1 week, meaning every week there
+will be a new version of the Relay web app on the [Production][prod]
+environment. To do this, we first release code to [Dev][dev] and
+[Stage][stage].
+
+## Release to Dev
+~~Every commit to `main` is automatically deployed to the [Dev][dev] server.~~
+(Since the [Great GitHub Heroku Incident of 2022][github-heroku-incident], we
+have disabled GitHub integration on our Heroku app.)
+We try to push `main` to [Dev][dev] often. To do so, you need to add the heroku
+app as a remote:
+
+* `heroku git:remote -a fx-private-relay`
+
+Then, you can push your local `main` to heroku:
+
+* `git push heroku main`
+
+We can also push un-merged branches:
+
+* `get push heroku change-1:main`
+
+
+## Release to Stage
+Every tag pushed to GitHub is automatically deployed to the [Stage][stage]
+server. The standard practice is to create a tag from `main` every Tuesday at
+the end of the day, and to name the tag with `YYYY-MM-DD` [CalVer][calver]
+syntax. This tag will include only the changes that have been merged to `main`.
+E.g., the following `2022.08.02` tag includes only `change-1` and `change-2`.
+
+```mermaid
+%%{init: { 'theme': 'base', 'gitGraph': {'rotateCommitLabel': true} } }%%
+    gitGraph
+       commit
+       branch change-1
+       commit
+       commit
+       checkout main
+       branch change-2
+       commit
+       checkout main
+       merge change-1
+       branch change-3
+       commit
+       commit
+       checkout main
+       merge change-2 tag: "2022.08.02"
+       checkout change-3
+       commit
+       commit
+       checkout main
+```
+
+## Release to Prod
+We leave the tag on [Stage][stage] for a week so that we (and especially QA)
+can check the tag on GCP infrastucture before we deploy it to production. To
+deploy the tag to production:
+
+1. [Make a release on GitHub][github-new-release] for the tag.
```

Comment:
I suggest making a release, marked as a pre-release, when you make the initial tag, and then switch off pre-release for tags that go to prod.